.azure-pipelines.yml

```diff
@@ -0,0 +1,18 @@
+
+# https://github.com/Microsoft/azure-pipelines-image-generation/blob/master/images/macos/
+variables:
+  VM_IMAGE: macOS-10.15
+  DEFAULT_NODE_VERSION: "12.x"
+
+pool:
+  vmImage: "$(VM_IMAGE)"
+
+stages:
+- stage: Unit_Tests_And_Linters
+  jobs:
+  - job: Node_Unit_Tests
+    steps:
+    - template: azure-templates/node_setup_steps.yml
+    - script: npm install
+    - script:
+        npm run test
```

Comment:
should we also run `lint` task or it is already included?

Reply:
The output had `lint`, so this should have